pkg/test/framework/environment/environment.go

```diff
@@ -110,6 +110,9 @@ type (
 	Implementation interface {
 		// EnvironmentID is the unique ID of the implemented environment.
 		EnvironmentID() settings.EnvironmentID
+
+		// Evaluate the given template with environment specific template variables.
+		Evaluate(tmpl string) (string, error)
```

Comment:
Just a note: I suspect we'll have to think a bit about this ... we need a different approach here going forward. The tests shouldn't have environment-specific things in them.